cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeConfigurationBuilder.java

```diff
@@ -0,0 +1,342 @@
+
+    private final StringWriter stringWriter = new StringWriter();
+    private final PrintWriter writer = new PrintWriter(stringWriter);
+
+    private String securityProtocol = "PLAINTEXT";
+
+    /**
+     * Bridge configuration template constructor
+     *
+     * @param bridgeId  the bridge ID
+     * @param bootstrapServers  Kafka cluster bootstrap servers to connect to
+     */
+    public KafkaBridgeConfigurationBuilder(String bridgeId, String bootstrapServers) {
+        printHeader();
+        configureBridgeId(bridgeId);
+        configureBootstrapServers(bootstrapServers);
+    }
+
+    /**
+     * Renders the bridge ID configurations
+     *
+     * @param bridgeId  the bridge ID
+     */
+    private void configureBridgeId(String bridgeId)   {
+        printSectionHeader("Bridge ID");
+        writer.println("bridge.id=" + bridgeId);
+        writer.println();
+    }
+
+    /**
+     * Renders the Apache Kafka bootstrap servers configuration
+     *
+     * @param bootstrapServers  Kafka cluster bootstrap servers to connect to
+     */
+    private void configureBootstrapServers(String bootstrapServers) {
+        printSectionHeader("Kafka bootstrap servers");
+        writer.println("kafka.bootstrap.servers=" + bootstrapServers);
+        writer.println();
+    }
+
+    /**
+     * Configure the Kafka security protocol to be used
+     * This internal method is used when the configuration is build, because the security protocol depends on
+     * TLS and SASL authentication configurations and if they are set
+     */
+    private void configureSecurityProtocol() {
+        printSectionHeader("Kafka Security protocol");
+        writer.println("kafka.security.protocol=" + securityProtocol);
+    }
+
+    /**
+     * Configures the Kafka config providers used for loading some parameters from env vars and files
+     * (i.e. user and password for authentication)
+     *
+     * @return  the builder instance
+     */
+    public KafkaBridgeConfigurationBuilder withConfigProviders() {
+        printSectionHeader("Config providers");
+        writer.println("kafka.config.providers=strimzienv,strimzifile,strimzidir");
+        writer.println("kafka.config.providers.strimzienv.class=org.apache.kafka.common.config.provider.EnvVarConfigProvider");
+        writer.println("kafka.config.providers.strimzienv.param.allowlist.pattern=.*");
+        writer.println("kafka.config.providers.strimzifile.class=org.apache.kafka.common.config.provider.FileConfigProvider");
+        writer.println("kafka.config.providers.strimzifile.param.allowed.paths=/opt/strimzi");
+        writer.println("kafka.config.providers.strimzidir.class=org.apache.kafka.common.config.provider.DirectoryConfigProvider");
+        writer.println("kafka.config.providers.strimzidir.param.allowed.paths=/opt/strimzi");
+        writer.println();
+        return this;
+    }
```

Comment:
If we don't allow users to configure the providers, it should be mentioned in the release notes as that is potentially backwards incompatible. But maybe we can simply handle it and just add our own as we do for broker?

Reply:
Yeah I changed to add the support as we have for broker.